src/platform/user/profile/vet360/components/PhoneField/PhoneEditModal.jsx

```diff
@@ -72,19 +71,22 @@ class PhoneEditModal extends React.Component {
     let defaultFieldValue;
 
     if (this.props.data) {
-      defaultFieldValue = merge(this.props.data, {
-        inputPhoneNumber:
-          this.props.data &&
-          [this.props.data.areaCode, this.props.data.phoneNumber].join(''),
+      defaultFieldValue = {
+        ...this.props.data,
+        inputPhoneNumber: `${this.props.data.areaCode}${
+          this.props.data.phoneNumber
+        }`,
+        extension: this.props.data.extension || '',
```

Comment:
This is the fix: if an extension is not set it comes back as `null` from VA Profile, which breaks in the new SchemaForm-based modal. So use an empty string instead and everything will be happy.

Reply:
I'd prefer to use a nullish coalescing operator here but apparently our build doesn't support that. I coulda swore we added support for that when we added optional chaining. I even told someone to use it in a code review a few weeks ago 😞

Reply:
Oh no! :sweat_smile: 

I remember talking about it a while back, but I don't remember whether or not we added it. I personally can't think of a reason it would be bad. :man_shrugging: